modules/logging-upgrading-clo.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-upgrading.adoc
+
+:_content-type: PROCEDURE
+[id="logging-upgrading-clo_{context}"]
+= Updating the Cluster Logging Operator
+
+To update the Cluster Logging Operator to a new major release version, you must modify the update channel for the Operator subscription.
+
+.Prerequisites
+
+* You have installed the Red Hat OpenShift Logging Operator.
+* You have administrator permissions.
+* You have access to the {product-title} web console and are viewing the *Administrator* perspective.
+
+.Procedure
+
+. Navigate to *Operators* -> *Installed Operators*.
+
+. Select the *openshift-logging* project.
+
+. Click the *Red Hat OpenShift Logging* Operator.
+
+. Click *Subscription*. In the *Subscription details* section, click the *Update channel* link. This link text might be *stable* or *stable-5.x*, depending on your current update channel.
```

Comment:
nit: since we adopted 5.y.z notation, I feel we should make it consistent throughout the docs
```suggestion
. Click *Subscription*. In the *Subscription details* section, click the *Update channel* link. This link text might be *stable* or *stable-5.y*, depending on your current update channel.
```

Reply:
In the docs previously, we refer to *stable-5.x* as the channel, but we updated the section in this PR that explains that there are major and minor updates to use *5.y.z* and *5.y* because this is actually how we refer to the releases ourselves, rather than using *x*. So, here Joao is just asking to make that consistent, which I actually think is a good idea, i.e. use *stable-5.y* here instead of *stable-5.x* - it doesn't really make a difference to the meaning of the content though, but I will update it for consistency unless anyone has major objections.

Side note, I am working on collating / updating the install docs that we have at the minute as part of the restructure, so any other places where *stable-5.x* is mentioned in the docs will be updated as part of that (outside of this PR).

Reply:
Updated